src/SB/Game/zEntDestructObj.cpp

```diff
@@ -48,141 +53,176 @@ void zEntDestructObj_FindFX()
     sEmitWeb = zParEmitterFind("PzREMIT_DOBJ_WEB");
     sEmitFire = zParEmitterFind("PAREMIT_FIRE");
     sEmitSmoke = zParEmitterFind("PAREMIT_FIRESMOKE");
-    sShrapDefault = (zParEmitter*) xSTFindAsset(xStrHash("destruct_obj_shrapnel"), NULL);
+    sShrapDefault = (zParEmitter*)xSTFindAsset(xStrHash("destruct_obj_shrapnel"), NULL);
 }
 
 void zEntDestructObj_Init(void* ent, void* asset)
 {
-    zEntDestructObj_Init((zEntDestructObj*) ent, (xEntAsset*) asset);
+    zEntDestructObj_Init((zEntDestructObj*)ent, (xEntAsset*)asset);
 }
 
 void zEntDestructObj_Init(zEntDestructObj* ent, xEntAsset* asset)
 {
-    zEntDestructObjAsset *dObjAsset;
-    xModelInstance *hitModelInst;
-    xModelInstance *destroyModelInst;
+    zEntDestructObjAsset* dObjAsset;
+    xModelInstance* hitModelInst;
+    xModelInstance* destroyModelInst;
     u32 shrapnelDestroyId;
     u32 shrapnelHitId;
     u32 sfxDestroyId;
     u32 sfxHitId;
     u32 hitModelId;
     u32 destroyModelId;
     U8 boundType;
-    xModelInstance *destroyModel;
-    xModelInstance *hitModel;
+    xModelInstance* destroyModel;
+    xModelInstance* hitModel;
+
+    zEntInit((zEnt*)ent, asset, 0x44535452U);
```

Comment:
this int literal should be a multicharacter literal.
`zEntInit((zEnt*)ent, asset, 'DSTR');` I think.